File: C++/I TECH CLASS/Encapsulation/basic.cpp
#include <iostream>
using namespace std;

class employee
{
    private:
    int salary;

    public:
    void setsalary(int s)
    {
        salary=s;
    }
    int getsalary()
    {
        return salary;
    }
};

int main() {

    employee e1;
    e1.setsalary(500);

    cout << "Salary is " << e1.getsalary();

return 0;
}

File: Java/Inheritance/Basic.java
package Java.Inheritance;

import java.util.Scanner;


class Employee
{
    double salary = 40000; 
}

public class Basic extends Employee {
    int bonus = 10000;
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Basic b = new Basic();
        System.out.println("Salary = "+b.salary);        
        System.out.println("Bonus = "+b.bonus);    
        sc.close();
        }
}
